<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="saveStatusButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>保存状态</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoSolveButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>自动求解</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rankBrowse">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>排名记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gameSettingButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>游戏设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>210</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>游戏开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undoStepButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>撤销</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitGameButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton0">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton8">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Adobe Arabic&quot;;</string>
    </property>
    <property name="text">
     <string>步数</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="gameTimeLCD">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>20</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Adobe Arabic&quot;;</string>
    </property>
    <property name="text">
     <string>用时</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="gameStepLCD">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>20</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
